python/src/aiconfig/editor/client/src/components/prompt/PromptContainer.tsx

```diff
@@ -190,7 +203,21 @@ export default memo(function PromptContainer({
 
           {prompt.outputs && prompt.outputs.length > 0 && (
             <>
+            <Flex justify="space-between" direction="column">
               <PromptOutputBar />
+              {!readOnly && prompt.outputs?.length && deleteOutput ? (
```

Comment:
Instead of adding the button here, can we add it to the prompt menu?

https://github.com/lastmile-ai/aiconfig/blob/main/python/src/aiconfig/editor/client/src/components/prompt/PromptMenuButton.tsx

I imagine we will have other capabilities like Copy/Duplicate prompt which will also be added in the future, so it will prevent too many buttons everywhere